tests/test_manager.py

```diff
@@ -10,6 +10,8 @@
 # License version 3 and under the 3-clause BSD license. For more
 # information, see LICENSE.AGPL and LICENSE.BSD.
 """Unit tests for the :mod:`flask_restless.manager` module."""
+from json import loads
```

Comment:
Remove this and add the line `from .helpers import loads` below the line `from .helpers import force_content_type_json`. (This loads `flask.json.loads` instead of `json.loads`.)